_dashboards/index.md

```diff
@@ -15,16 +15,16 @@ redirect_from:
 
 OpenSearch Dashboards is an open-source, integrated visualization tool that makes it easy for users to explore their data in OpenSearch. From real-time application monitoring, threat detection, and incident management to personalized search, OpenSearch Dashboards gives you the data visualizations needed to graphically represent trends, outliers, and patterns in your data.
 
-##### Example: Data visualizations in OpenSearch Dashboards  
+**Example: Data visualizations in OpenSearch Dashboards**  
 
 <img src="{{site.url}}{{site.baseurl}}/images/dashboard-flight.png" alt="User interface showing several data visualizations">
 
-# Use cases for OpenSearch Dashboards
+## Use cases for OpenSearch Dashboards
 
 In OpenSearch Dashboards, a data dashboard is a collection of charts, graphs, and insights that give you a snapshot of the data that you're interested in and that you interact with. 
```

Comment:
I would remove the word "insights" - I'm not sure what's meant by it here.

Reply:
Revised for clarity.